lib/optimizely_server_side/helpers/support.rb

```diff
@@ -24,17 +24,41 @@ module Support
     def experiment(experiment_key, options = {}, &blk)
       # Merge any other options to user attributes
       OptimizelyServerSide.configuration.user_attributes.merge!(options.stringify_keys)
-      variation_key = optimizely_sdk_project_instance(experiment_key)
+      variation_key = optimizely_sdk_project_instance(experiment_key, "experiment")
       OptimizelyServerSide::Experiment.new(experiment_key, variation_key).start(&blk)
     end
     alias_method :feature_flip, :experiment
 
-    def optimizely_sdk_project_instance(experiment_key)
+    def optimizely_sdk_project_instance(experiment_key, type)
+      case type
+      when "experiment"
+        get_optimizely_instance.activate(experiment_key, visitor_id,
+                OptimizelyServerSide.configuration.user_attributes.reject { |k,v| k == 'visitor_id'})
+      when "feature"
+        get_optimizely_instance.is_feature_enabled(experiment_key, visitor_id,
+                OptimizelyServerSide.configuration.user_attributes.reject { |k,v| k == 'visitor_id'})
+      end  
+    end
+
+    def feature_test(experiment_key, options = {}, &blk)
+      OptimizelyServerSide.configuration.user_attributes.merge!(options.stringify_keys)
+      variation_key = optimizely_sdk_project_instance(experiment_key, "feature")
+    end
+    
+    [:string, :boolean, :integer, :double].each do |type|
+      define_method :"get_feature_variable_#{type}" do |experiment, variable_name |
```

Comment:
Can you point me to Optimizely SDK doc on this? Would like to know what it's doing.

Reply:
Here you go: https://docs.developers.optimizely.com/full-stack/docs/variables